crudApp/src/components/Edit.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from "react";
+import { Button, Form } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import { useDispatch } from "react-redux";
+
+import { editEmp } from "../actions";
+import Employees from "./Employees";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+function Edit() {
+  const dispatch = useDispatch();
+  const [name, setName] = useState("");
+  const [age, setAge] = useState("");
+  const [id, setId] = useState("");
+  let history = useNavigate();
+  var index = Employees.map((e) => {
```

Comment:
if you want this index variable inside the function only then move it in function's scope. also use const to define it dont use var